SECURITY.md

```diff
@@ -20,6 +20,8 @@ We kindly ask you to refrain from malicious acts that put our users, the project
 
 ## Reporting a security issue
 
+> Please, do not use the provided email address to report issues which are not related to security vulnerabilities
```

Comment:
With and without comma is correct, not sure if we want to emphasize "please" or remove the comma, which adds some small break when you speak it like "please ... do not" while the other is more like "please do not" which is "more" correct.

```suggestion
> Please do not use the provided email address to report issues which are not related to security vulnerabilities
```

Reply:
Agree, I actually want to emphasize that. The mailbox is getting noisy.